lib/ipmi_client.robot

```diff
@@ -20,6 +20,7 @@ Run IPMI Command
 Run IPMI Standard Command
     [arguments]    ${args}
     ${output}=  Execute Command    /tmp/ipmitool -I dbus ${args}
+    Return From Keyword If    '${output}' == ''    error
```

Comment:
output being empty does not mean that your command really has failed. Instead I would add a code to handle the error based on the exit status(return code) for the command. 

I would consider playing around these arguments to the execute commands -  return_stdout=True, return_stderr=True, return_rc=True and then check the rc value.